networkx/algorithms/isomorphism/tests/test_vf2pp.py

```diff
@@ -1571,39 +1608,49 @@ def test_wikipedia_graph(self):
     def test_non_isomorphic_same_degree_sequence(self):
         r"""
                 G1                           G2
-        x--------------x              x--------------x
-        | \            |              | \            |
-        |  x-------x   |              |  x-------x   |
-        |  |       |   |              |  |       |   |
-        |  x-------x   |              |  x-------x   |
-        | /            |              |            \ |
-        x--------------x              x--------------x
+        1-------------2              1-------------2
+        | \           |              | \           |
+        |  5-------6  |              |  5-------6  |
+        |  |       |  |              |  |       |  |
+        |  8-------7  |              |  8-------7  |
+        | /           |              |           \ |
+        4-------------3              4-------------3
         """
         edges1 = [
             (1, 5),
             (1, 2),
             (4, 1),
             (3, 2),
             (3, 4),
-            (4, 8),
             (5, 8),
             (6, 5),
             (6, 7),
             (7, 8),
+            (4, 8),
```

Comment:
Out of curiosity, why this change? I see you did the same thing higher up.

Reply:
I am trying to make the edges1 and edges2 lists more readable. 
They are now the same up to the last edge.
The pictures show what is going on but comparing the lists is helpful too.